### DISCUSSION TITLE
Blank page Cloudflare

### DISCUSSION THREAD

**Discussion by barcelona92** [benbusby/whoogle-search#333] [General]
Hi,

When put a DNS Proxy activate in Clodflare, appear Blank Page.

Refused to load the script 'https://ajax.cloudflare.com/cdn-cgi/scripts/7du00a10a/cloudflare-static/rocket-loader.min.js' because it violates the following Content Security Policy directive: "script-src 'self'". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.

Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-3sWwXpWFS'), or a nonce ('nonce-...') is required to enable inline execution.

Any idea?

In local i work perfectly http://IP:5600  work perfectly.

The Whoogle is installed in docker and Nginx Proxy Manager,

**Comment by alefvanoon** [comment]
Hi!
Still you have this problem?
Hmmm, my instance working great on Cloudflare!
checkout https://s.alefvanoon.xyz
